release-notes/v5.5.11.md

```diff
@@ -0,0 +1,4 @@
+#### <sub><sup><a name="v5511-note-1" href="#5511-note-1">:link:</a></sup></sub> feature
+
+* Add loading indicator on dashboard while awaiting API responses. #5427
```

Comment:
From what I got reproducing locally, this is *only* for the first API request for the dashboard view - do you think that should be emphasized here?

Reply:
Good point! Updated the release note